README.md

```diff
@@ -1 +1,52 @@
-# clojure-workshop
\ No newline at end of file
+# Clojure Workshop
+
+Basic workshop/course on Clojure programming.
+
+## Prerequisite
+
+##### Java
+Version 1.8.0 or higher
+The command: `java -version` should output: `[...] version "1.8.0_202"`
```

Comment:
For longevity of this file it's probably more suitable to have `1.8.0_xxx`, since we don't care about versions as long as they fall within `1.8.0`, right?

Reply:
Very good point! Will change